tick/linear_model/model_logreg.py

```diff
@@ -119,8 +132,10 @@ def sigmoid(coeffs: np.ndarray, out: np.ndarray = None) -> np.ndarray:
             ``out``
         """
         if out is None:
-            out = np.empty(coeffs.shape[0])
-        _ModelLogReg.sigmoid(coeffs, out)
+            out = np.empty(coeffs.shape[0], dtype=coeffs.dtype)
+        # this following line requires "np.dtype('floatxx')
+        #  for reasons unknown
```

Comment:
we can't use it because it's a static function and there is no "self"

at this point it should be impossible for coeffs.dtype to be anything other than "float32" or "float64"

so I can't see an error occurring from a missing key in the map